README.md

```diff
@@ -128,9 +128,17 @@ The following are valid query attributes:
 
 | Attribute | Description  |
 |---|---|
-| `id` | The unique ID that is associated with each dataset. |
-| `contents` | A client can perform keyword queries with this attribute. |
-| `media_type` | The media (MIME) type of the IRM. |
+| `id` | The unique ID that is associated with each Dataset |
```

Comment:
❓ Should we also be documenting the types of these attributes?

Reply:
That is a good question. Should we? My opinion, now that I think about it, is "no".

Reply:
Since the CST are our public API, it needs to be documented somewhere. I don't know if a README is the best place for it.

Reply:
I don't think searching on DatasetId is part of the CST

Reply:
"content" is not part of the CST either. I left it because our test cases depend on it. Even manual testing depends on it.